Watchman.Discord/Areas/Administration/Controllers/AdministrationController.cs

```diff
@@ -48,17 +49,17 @@ public AdministrationController(IQueryBus queryBus, UsersService usersService, D
         }
 
         [AdminCommand]
-        [DiscordCommand("messages")]
-        public async Task ReadUserMessages(DiscordRequest request, Contexts contexts)
+        public async Task ReadUserMessages(MessagesCommand command, Contexts contexts)
         {
-            var mention = request.GetMention();
-            var selectedUser = await this._usersService.GetUserByMentionAsync(contexts.Server, mention);
+            var messagesService = this._messagesServiceFactory.Create(contexts);
+            var selectedUser = await this._usersService.GetUserByIdAsync(contexts.Server, command.User);
             if (selectedUser == null)
             {
-                throw new UserNotFoundException(mention);
+                await messagesService.SendResponse(x => x.InvalidArguments());
+                return;
             }
+            var timeRange = TimeRange.ToNow(DateTime.UtcNow - command.Time);
```

Comment:
```suggestion
            var timeRange = TimeRange.ToNow(DateTime.Now - command.Time);
```
Ostatnio przeszliśmy jednak z UtcNow na Now, bo DateTime w miarę dobrze sobie z tym radzi i na razie nie było problemów z opieraniem się na lokalnym czasie

Reply:
+ dopisz komentarz //todo: zmienić DateTime.Now na Contexts.SentAt
(na razie nie mamy jeszcze SentAt w contexts, ale jest na to task)